Latest Botan3 moves secure_scrub_memory into mem_ops.h

*(Note: I already have a fix, I just wanted an issue to reference)*

## Overview

1. Build and install the latest botan3 library from https://github.com/randombit/botan
2. Attempt to build KeepassXC

## Expected Behavior
Expected KeepassXC to finish building successfully.

## Actual Behavior
Fails to build from source with error message `error: 'secure_scrub_memory' is not a member of 'Botan'`:
<details>
<summary>Tail of CMake build log (~26 lines)</summary>

```
[ 93%] Building CXX object src/CMakeFiles/keepassx_core.dir/gui/reports/ReportsPageBrowserStatistics.cpp.o
[ 93%] Building CXX object src/CMakeFiles/keepassx_core.dir/gui/reports/ReportsWidgetBrowserStatistics.cpp.o
[ 93%] Building CXX object src/CMakeFiles/keepassx_core.dir/keys/drivers/YubiKey.cpp.o
[ 94%] Building CXX object src/CMakeFiles/keepassx_core.dir/keys/drivers/YubiKeyInterface.cpp.o
[ 94%] Building CXX object src/CMakeFiles/keepassx_core.dir/keys/drivers/YubiKeyInterfaceUSB.cpp.o
/tmp/git/keepassxc/src/keys/FileKey.cpp: In static member function 'static void FileKey::createXMLv2(QIODevice*, int)':
/tmp/git/keepassxc/src/keys/FileKey.cpp:246:12: error: 'secure_scrub_memory' is not a member of 'Botan'
  246 |     Botan::secure_scrub_memory(key.data(), static_cast<std::size_t>(key.capacity()));
      |            ^~~~~~~~~~~~~~~~~~~
/tmp/git/keepassxc/src/keys/FileKey.cpp: In member function 'bool FileKey::loadXml(QIODevice*, QString*)':
/tmp/git/keepassxc/src/keys/FileKey.cpp:369:12: error: 'secure_scrub_memory' is not a member of 'Botan'
  369 |     Botan::secure_scrub_memory(keyFileData.data.data(), static_cast<std::size_t>(keyFileData.data.capacity()));
      |            ^~~~~~~~~~~~~~~~~~~
/tmp/git/keepassxc/src/keys/FileKey.cpp: In member function 'bool FileKey::loadHex(QIODevice*)':
/tmp/git/keepassxc/src/keys/FileKey.cpp:425:12: error: 'secure_scrub_memory' is not a member of 'Botan'
  425 |     Botan::secure_scrub_memory(data.data(), static_cast<std::size_t>(data.capacity()));
      |            ^~~~~~~~~~~~~~~~~~~
/tmp/git/keepassxc/src/keys/FileKey.cpp: In member function 'bool FileKey::loadHashed(QIODevice*)':
/tmp/git/keepassxc/src/keys/FileKey.cpp:451:12: error: 'secure_scrub_memory' is not a member of 'Botan'
  451 |     Botan::secure_scrub_memory(buffer.data(), static_cast<std::size_t>(buffer.capacity()));
      |            ^~~~~~~~~~~~~~~~~~~
make[2]: *** [src/CMakeFiles/keepassx_core.dir/build.make:2425: src/CMakeFiles/keepassx_core.dir/keys/FileKey.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:473: src/CMakeFiles/keepassx_core.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
make: Leaving directory '/usr/src/git/keepassxc/_build'
```
</details>

## Context
Revisions:
- a3717c7acd87d5e34439aa1f9f17db767a6470f3 (HEAD of `develop`
- dd21defcf38e06e6aa4286fe85bfbb399b1e36e9 (HEAD of `release/2.7.x`

OS: Linux x86_64

--- a/src/keys/FileKey.h
+++ b/src/keys/FileKey.h
@@ -19,6 +19,7 @@
 #ifndef KEEPASSX_FILEKEY_H
 #define KEEPASSX_FILEKEY_H
 
+#include <botan/mem_ops.h>
 #include <botan/secmem.h>
 
 #include "keys/Key.h"

